com.nomachine.nxplayer.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Encoding=UTF-8
+Comment=Access NoMachine computers over the network
+Exec="/app/extra/NX/bin/nxplayer"
```

Comment:
why isn't this the same as `command` in the manifest?

Reply:
Oversight while making changes to how it is executed before PR creation. It is now corrected.